recipes/simbody/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conan import ConanFile, tools
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain
+from conan.tools.scm import Version
+import os
+import textwrap
+
+
```

Comment:
```suggestion
required_conan_version = ">=1.50.0"

```
Need to check a minimal version to be used.